ui/StatusQ/src/StatusQ/Components/StatusItemSelector.qml

```diff
@@ -210,7 +210,7 @@ Rectangle {
 
                         asset.height: root.asset.height
                         asset.width: root.asset.width
-                        asset.name: root.useLetterIdenticons ? model.text : model.imageSource
+                        asset.name: root.useLetterIdenticons ? model.text : (model.imageSource ?? "")
```

Comment:
This needs 2.15

Reply:
Really? It works fine... Imports are for libs (and components in them) and have nothing to do with js version, from what I understand.

Reply:
You are probably correct; just a habit of mine I think :) Do as you think is best

Reply:
Regarding import version two approaches come to my mind:
- use script to update everything to *.15 in a one shot
- ignore it for now bc finally we will get rid of version totally in qt 6.x

not sure which one is better